Structure_Engine/Query/Description.cs

```diff
@@ -355,13 +355,40 @@ public static string Description(this Constraint6DOF constraint)
         /***************************************************/
 
         [Description("Generates a default description for the BarRelease as 'StartReleaseNameOrDesc - EndReleaseNameOrDesc'.")]
-        [Input("release", "The Constraint6DOF to get a default description for.")]
-        [Output("desc", "The generated description for the constraint.")]
+        [Input("release", "The BarRelease to get a default description for.")]
+        [Output("desc", "The generated description for the BarRelease.")]
         public static string Description(this BarRelease release)
         {
             return release.StartRelease.DescriptionOrName() + " - " + release.EndRelease.DescriptionOrName();
         }
 
+        /***************************************************/
+
+        [Description("Generates a default description for the LinkConstraint based on its fixities where x denoted fixed and f denotes free.")]
+        [Input("release", "The LinkConstraint to get a default description for.")]
```

Comment:
CheckCompliance will flag this up, but I'll comment here to highlight it for you @IsakNaslundBh - input attribute name doesn't match the method argument name.

Reply:
🤦

Reply:
❤️